rtrlib/lib/lrtr_vrf.h

```diff
@@ -0,0 +1,19 @@
+#ifndef LRTR_UTILS_VRF_H
+#define LRTR_UTILS_VRF_H
+
+#include <sys/capability.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <stdbool.h>
+
+/* apis used by caller of API */
+void lrtr_vrf_init(void);
+void lrtr_vrf_init_capabilities(cap_value_t **cap_admin, cap_t *cap_user, int num,
+                                uid_t *uid_admin, uid_t *uid_user,
+                                bool netns_mode);
+
+/* internal api. used by transport.c */
```

Comment:
Internal API functions should be in a separate header suffixed with "_private"

Reply:
I will work on this tomorrow